pkg/storage/registry/dynamic/dynamic.go

```diff
@@ -217,13 +217,10 @@ func (d *dynamic) FindProviders(ctx context.Context, ref *provider.Reference) ([
 				Address:      address,
 			})
 		} else {
-			err := errtypes.InternalError("storage provider address not configured for mountID " + p.ProviderId)
-			l.Error().Err(err).Send()
-			return nil, err
+			l.Error().Str("mount", p.ProviderId).Msg("Storage provider address not configured for mount, skipping")
 		}
 	}
 
-	l.Debug().Msgf("resolved storage providers %+v", providers)
-
+	l.Info().Msgf("resolved storage providers %+v", providers)
```

Comment:
Maybe info is too much for this?
```suggestion
	l.Debug().Msgf("resolved storage providers %+v", providers)
```

Reply:
(in fact the `ref` is already logged by how the logger was initialized, so I've just reverted to Debug)